indico/modules/events/papers/models/call_for_papers.py

```diff
@@ -51,6 +52,10 @@ def has_ended(self):
     def is_open(self):
         return self.has_started and not self.has_ended
 
+    def can_submit_papers(self, user):
```

Comment:
Probably not needed; it's only used to implement the outside-deadlines-submission ACL in abstracts but we don't have this in papers (at least not yet and not in legacy)